handbook/engineering/README.md

```diff
@@ -47,7 +47,7 @@ If there are no product changes, and the DRI decides to prioritize the story, th
 ### Fix a bug
```

Comment:
```suggestion
### Fix a bug

All bug fix pull requests should have a mention back to the issue they resolve with #<ticket_number> in the description or even in a comment. Please do not use any [automated words](https://docs.github.com/en/issues/tracking-your-work-with-issues/linking-a-pull-request-to-an-issue#linking-a-pull-request-to-an-issue-using-a-keyword) since we don't want the tickets auto-closing when PR's are merged. 
```